wasmbinding/bindings/msg.go

```diff
@@ -223,3 +226,25 @@ type ResubmitFailure struct {
 type ResubmitFailureResponse struct {
 	FailureId uint64 `json:"failure_id"`
 }
+
+type Dex struct {
+	Deposit                  *types.MsgDeposit                  `json:"deposit"`
+	Withdrawal               *types.MsgWithdrawal               `json:"withdrawal"`
+	PlaceLimitOrder          *MsgPlaceLimitOrder                `json:"place_limit_order"`
+	WithdrawFilledLimitOrder *types.MsgWithdrawFilledLimitOrder `json:"withdraw_filled_limit_order"`
+	CancelLimitOrder         *types.MsgCancelLimitOrder         `json:"cancel_limit_order"`
+	MultiHopSwap             *types.MsgMultiHopSwap             `json:"multi_hop_swap"`
+}
+
+type MsgPlaceLimitOrder struct {
+	Creator          string               `json:"creator,omitempty"`
+	Receiver         string               `json:"receiver,omitempty"`
+	TokenIn          string               `json:"token_in,omitempty"`
+	TokenOut         string               `json:"token_out,omitempty"`
+	TickIndexInToOut int64                `json:"tick_index_in_to_out,omitempty"`
+	AmountIn         math.Int             `json:"amount_in"`
+	OrderType        types.LimitOrderType `json:"order_type,omitempty"`
```

Comment:
let's consider having the `OrderType` type as string and then handled the same way as here: https://github.com/neutron-org/neutron/blob/e605ed3db4381994ee8185ba4a0ff0877d34e67f/x/dex/client/cli/tx_place_limit_order.go#L47

reasoning is raw `types.LimitOrderType` at user's side is represented as numbers (0, 1, 2, 3 or 4, [proof](https://github.com/neutron-org/neutron-integration-tests/blob/e71b943c0d6f90a9fc869c6c1f01eca364c1d2f7/src/testcases/run_in_band/dex.test.ts#L297)). would be better to expect concrete string values and compare them to the enumerated string values